client/my-sites/theme/main.jsx

```diff
@@ -551,8 +568,8 @@ export default connect(
 		const backPath = getBackPath( state );
 		const currentUserId = getCurrentUserId( state );
 		const isCurrentUserPaid = isUserPaid( state, currentUserId );
-		const themeDetails = getThemeDetails( state, id );
-
+		const theme = selectedSite ? getTheme( state, selectedSite.ID, id ) : getTheme( state, 'wpcom', id );
+		const themeDetails = theme ? themeDetailsFromTheme( theme ) : {};
```

Comment:
I actually like having attrs encapsulated inside a `theme` object, so maybe we should change the component instead to read attrs from a `theme` prop?